in_toto/gpg/constants.py

```diff
@@ -27,6 +27,7 @@
 PACKET_TYPES = {
     'signature_packet': 0x02,
     'main_pubkey_packet': 0x06,
+    'pub_subkey_packet': 0x0E,
```

Comment:
Are these arbitrarily chosen hexadecimal integers, or do they match the ones defined in RFC 4880?

Reply:
I'll add a pointer to the RFC.